### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by minwoox**
- Caffeine 2.8.2 -> 2.8.4
- grpc-kotlin-stub 0.1.1 -> 0.1.2
- io.dropwizard.metrics 4.1.7 -> 4.1.9
- Jetty 9.4.28.v20200408 -> 9.4.29.v20200521
- kotlinx-coroutines-core 1.3.6 -> 1.3.7
- reactor-grpc 1.0.0 -> 1.0.1
- Retrofit 2.8.1 -> 2.9.0
- Rxjava3 3.0.3 -> 3.0.4
- Build
  - Awaitility 4.0.2 -> 4.0.3
  - Eureka 1.9.20 -> 1.9.22
  - Gradle 6.4 -> 6.4.1
  - gradle-node-plugin 2.2.3 -> 2.2.4

### CODE REVIEW

**Comment by ikhoon** [comment]
Let me handle this issue, once this PR is merged.
https://github.com/line/armeria/blob/b1decb5d0d33122ab4e42a06a0230cbc2f79249c/grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java#L97-L98

**Comment by minwoox** [comment]
> Let me handle this issue, once this PR is merged.

I forgot about it. I can change it in this PR. 😄

**Comment by ikhoon** [comment]
> > Let me handle this issue, once this PR is merged.
> 
> I forgot about it. I can change it in this PR. 😄

Thanks! @minwoox 🥰

File: grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java [hunk 1]
```diff
@@ -216,4 +238,29 @@ private static IllegalArgumentException newUnknownClientTypeException(Class<?> c
         }
         return ImmutableList.copyOf(descriptor.getMethods());
     }
+
+    private static Class<?> generatedStub(Annotation annotation, Method valueMethod) {
+        try {
+            final Object kClass = valueMethod.invoke(annotation, null);
+            return Class.forName(kClass.toString().substring("class ".length()));
```
Comment (ikhoon) [inline grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java:245]: How about casting `Object` to `Class<?>`?
```suggestion
            final Class<?> kClass = (Class<?>) valueMethod.invoke(annotation, null);
            return Class.forName(kClass.getName());
```

Comment (trustin) [reply grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java:245]: Why not just returning `kClass` then? No point of doing `Class.forName()`

Comment (ikhoon) [reply grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java:245]: > Why not just returning `kClass` then? No point of doing `Class.forName()`

Ahh, you are right! 🤣

Comment (minwoox) [reply grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcClientFactory.java:245]: Oops fixed. 😅

**Pull request merged**